SetMultiScreensVideoMode.brs

```diff
@@ -0,0 +1,40 @@
+' Multi Screens VideoMode Plugin Example
+' Plugin function name - setvideomode
+' Plugin return type - As Array: For series5 players, it is plugin's responsibility to 
+' return the same format as GetScreenModes() returns. Then the presentation autorun will
+' pick up your ScreenConfigList and set video modes for your screens.
+' Note: BrightAuthor:connected always sets :fullres on all of the videomodes for the XC5
+Function setvideomode(videoModeInputs, bsp)
+
+	print "setvideomode_ForMultiScreens"
+
+	vm = CreateObject("roVideoMode")
+	sm = vm.GetScreenModes()
+
+	sm[0].video_mode="3840x2160x29.97p:rblank:fullres"

```

Comment:
Could you add some comments to give more instructions for the following:

- Where are the areas that user should modify
- There is a field "name" that is important. ATM, it's default by order 0->hdmi1, 1->hdmi2, etc. But any script can change to order. It's safer to include name in the example to prevent mismatch and unexpected behavior
- Clarify that number of array items depend on the number of hdmi outputs. e.g. XC2055 has 3 - hdmi1, hdmi2, typeC
- if want to turn an output to off, set enabled = false is enough. No need to add the other fields.
- if do not want to touch the existing settings of a screen, can skip setting it.

Reply:
Details doc instructions page added. Hope that solves your ask.

Reply:
It doesn't look like covers everything.

- Where are the areas that user should modify => suggest have a comment(e.g. ' TODO) at the beginning of where user should touch

- There is a field "name" that is important. ATM, it's default by order 0->hdmi1, 1->hdmi2, etc. But any script can change to order. It's safer to include name in the example to prevent mismatch and unexpected behavior => Can we include "name" in the example here?

- Clarify that number of array items depend on the number of hdmi outputs. e.g. XC2055 has 3 - hdmi1, hdmi2, typeC => Doesn't look like the docs is clear, it only has example for XC4055.

- if want to turn an output to off, set enabled = false is enough. No need to add the other fields. => This is mentioned in the doc

- if do not want to touch the existing settings of a screen, can skip setting it. => not mentioned in the doc

Reply:
> Can we include "name" in the example here?
This one confuses me a little since we don't have this field included in the docs page, and the plugin was written based on our current docs.

So it sounds to me that you are suggesting a few clarifications on MultiScreen Configuration Docs or roVideoMode, which those detailed info/asks can be a DOCS ticket. I do not feel like it is best practice to document all those info in the Plugin. Would that make sense to create a DOCS ticket? @mingjun7 @bhastings1019

Reply:
@mxin512 name is on the docs page. It's just not in the example. See
"name: On the XC platform, these are HDMI-1 through HDMI-4 but can differ across platforms."

I leave you guys to decide if it helps to add the "name" field. Just want to mention that the order can be different.

Sounds good. We can have a DOCS ticket to capture the details. I filed https://brightsign.atlassian.net/browse/DOCS-1165.

Reply:
Aha, okay. Thanks for the docs ticket. I think it makes sense to align the videomode plugin with our docs example.

Reply:
No problem. Stephanie has update the docs. Feel free to take a look.